lib/msal-core/src/Utils.ts

```diff
@@ -712,13 +712,22 @@ export class Utils {
     return paramsString;
   }
 
+  /**
+   * Check to see if there are SSO params set in the Request
+   * @param request
+   */
+  static isSSOParam(request: AuthenticationParameters) {
+      return request && (request.account || request.sid || request.loginHint);
+  }
+
   /**
    * Utility to test if valid prompt value is passed in the request
    * @param request
    */
-  static validateRequestParameters (request: AuthenticationParameters) {
-    if (!([PromptState.LOGIN, PromptState.SELECT_ACCOUNT, PromptState.CONSENT, PromptState.NONE].indexOf(request.prompt) >= 0)) {
-      console.log("Invalid Prompt Value");
+  static validatePromptParameter (prompt: string) {
+    if (!([PromptState.LOGIN, PromptState.SELECT_ACCOUNT, PromptState.CONSENT, PromptState.NONE].indexOf(prompt) >= 0)) {
```

Comment:
can this just be Object.keys(PromptState).indexOf(prompt >= 0)

Reply:
Actually yes. There was an error seen when I made some ErrorObject call and that will come in a different PR later. Will change it up then.

Reply:
Actually I have checked this, Object.keys(PromptState) keys off the key not the value. User passes "login" but this keys off LOGIN. There could be some other way to match this, resolving this for now. Will revisit when I add the error.